trulens_eval/trulens_eval/tests/groundedness_smoke_tests.ipynb

```diff
@@ -0,0 +1,224 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    test_cases_gen = generate_summeval_groundedness_golden_set("./datasets/summeval_test_100.json")
</pre><p>Can you print a few of these test cases just for illustration?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/527/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>np will do</p>